src/System.Security.Cryptography.Cng/src/System/Security/Cryptography/ECDsaCng.ImportExport.cs

```diff
@@ -0,0 +1,558 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Yes I was wondering how that would fall out once I port to desktop clr. I will get this corefx code base stable, port to clr and refactor as needed to share code, and then pull any of that refactoring again back again into corefx. I could try to do it all up front, but I will likely miss a few things.